index.d.ts

```diff
@@ -166,6 +166,12 @@ declare namespace fastifyMultipart {
        */
       files?: number;
 
+      /**
+       * For multipart forms, the max number of parts (fields + files)
+       * @default 1000
+       */
+      parts?: number;
```

Comment:
YOu can add the test here: https://github.com/fastify/fastify-multipart/blob/80a01f41c03cc29a074742438b70d95261bb56ff/test/index.test-d.ts#L24